CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to the `XgridInc/xc3` repository will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0] - 2023-05-11
+
+### Added
+- Initial release of the `XgridInc/xc3` repository.
+- `cloud_custodian_policies` folder which has the yaml for all the cloud custodian policies
+- `Pre-requirements` folder to refer the IAM Permission Set required to setup XC3
+- `Infrastructure` folder with the IaC necessary to provision XC3
+- `src` folder which has the code for all lambda functions
+- `tests` folder to test your IaC implementation and contribution
```

Comment:
make this testing version to v1.0.0

Reply:
Resolved in [bea9bfe](https://github.com/XgridInc/xc3/pull/8/commits/bea9bfefc5c3042b6a9437670d0ac2db8d12b89d)